HealthVault/Classes/Models/MHVDateTimeBase.m

```diff
@@ -21,6 +21,17 @@
 
 @implementation MHVDateTimeBase
 
+- (instancetype)init
```

Comment:
I was going to add an initWithDate: since Kayla needed it, if you want to add that too, that'd be nice :)